CHANGELOG.md

```diff
@@ -6,20 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## NEXT - TBD
```

Comment:
Should we mention release number here?

Reply:
good catch. I should've updated it. fixing